modules/menu/dynamic_menu.c

```diff
@@ -334,20 +382,22 @@ static int digit_handler(struct re_printf *pf, void *arg)
 /*Dynamic call menu*/
 static const struct cmd callcmdv[] = {
 
-{"aubitrate",    0,  CMD_PRM, "Set audio bitrate",    set_audio_bitrate    },
-{"audio_debug", 'A',       0, "Audio stream",         call_audio_debug     },
-{"callfind",     0,  CMD_PRM, "Find call <callid>",   cmd_find_call        },
-{"hold",        'x',       0, "Call hold",            cmd_call_hold        },
-{"line",        '@', CMD_PRM, "Set current call <line>", set_current_call  },
-{"mute",        'm',       0, "Call mute/un-mute",    call_mute            },
-{"reinvite",    'I',       0, "Send re-INVITE",       call_reinvite        },
-{"resume",      'X',       0, "Call resume",          cmd_call_resume      },
-{"sndcode",      0,  CMD_PRM, "Send Code",            send_code            },
-{"statmode",    'S',       0, "Statusmode toggle",    toggle_statmode      },
-{"transfer",    't', CMD_PRM, "Transfer call",        call_xfer            },
-{"video_debug", 'V',       0, "Video stream",         call_video_debug     },
-{"videodir",      0, CMD_PRM, "Set video direction",  set_video_dir        },
-{"medialdir",     0, CMD_PRM, "Set local media direction",  set_media_ldir },
+{"aubitrate",      0, CMD_PRM, "Set audio bitrate",        set_audio_bitrate },
+{"audio_debug",  'A',       0, "Audio stream",             call_audio_debug  },
+{"callfind",       0, CMD_PRM, "Find call <callid>",       cmd_find_call     },
+{"hold",         'x',       0, "Call hold",                cmd_call_hold     },
+{"line",         '@', CMD_PRM, "Set current call <line>",  set_current_call  },
+{"mute",         'm',       0, "Call mute/un-mute",        call_mute         },
+{"reinvite",     'I',       0, "Send re-INVITE",           call_reinvite     },
+{"resume",       'X',       0, "Call resume",              cmd_call_resume   },
+{"sndcode",        0, CMD_PRM, "Send Code",                send_code         },
+{"statmode",     'S',       0, "Statusmode toggle",        toggle_statmode   },
+{"transfer",     't', CMD_PRM, "Transfer call",            call_xfer         },
+{"att_transfer", 'T', CMD_PRM, "Attended call transfer",   call_att_xfer     },
+{"forw_transfer",'F',       0, "Forward attended call",    transfer_att_xfer },
```

Comment:
- Pls no underline symbols in command names!
- A lot of unrelated changed lines.

Reply:
I thought it is fine since there is `video_debug` and `audio_debug` already. Will remove them.

Reply:
Some already said the same to a prev PR of mine.